.circleci/config.yml

```
@@ -3,8 +3,49 @@ version: 2
 
 references:
   circleci_docker_primary: &circleci_docker_primary trussworks/circleci-docker-primary:tf12-5ee2cb35bc081dbc91da46afc3904ef0e1c111fb
+  postgres: &postgres postgres:11.6
+  db_migrate: &db_migrate ${AWS_ACCOUNT_ID}.dkr.ecr.us-west-2.amazonaws.com/easi-db-migrate:latest
 jobs:
   pre_test:
```

```suggestion
  build_db_migrate_image:
```
And update corresponding references...

---

I'm not very attached to this, but won't we have to change the name again if anything else gets added?

---

Possibly, but if renaming jobs makes their function more clear, then renaming them over time can be beneficial.